### DISCUSSION TITLE
wire:model not updated on input

### DISCUSSION THREAD

**Discussion by xfwil** [livewire/livewire#6930] [1. Help]
Hi, everyone !
Im new in here. I wanted ask about wire:model not updating on my input, like i have this form : 
```html
<form>
<input type="text" wire:model="name" />
...submitbtn
</form>
```
And this my livewire controller code : 
```php
class .....
{
public $name = "";

public function clearInput() 
{
$this->name = "";
}

public function save()
{
...code to store
...success
$this->clearInput();
}
```

The problem is that the name input is detected as empty, but there is still text inside it. It should also be empty. Can anyone help me? thank you !

Livewire version 3.0
Laravel 10

**Comment by Maarten-Dekker** [comment]
You can do the following to reset a field input:
`$this->reset('name')`

[https://livewire.laravel.com/docs/forms#resetting-form-fields](https://livewire.laravel.com/docs/forms#resetting-form-fields)

**Reply by xfwil** [reply]
Already tried for that syntax. But still not fixed yet

**Comment by danie-ramdhani** [comment]
i don't think you're using livewire since you say it "controller" word (because it called component here)

are you sure you in `app/Livewire/*.php` path?

**Reply by xfwil** [reply]
Yes, dont mind about "controller". I say this "controller" because i can control the livewire functions in here.

![image](https://github.com/livewire/livewire/assets/67111486/ec8e0412-5379-47e8-8913-0ee141d6da77)

**Comment by xfwil** [comment]
BUMP, maybe this bug? or template issues?

**Comment by PhiloNL** [comment]
@tokinaa please take a look at this example:
https://wirebox.app/b/qy82y

It will show you an example of clearing the input. Add a new item to the to-do list, and you will see that the input is reset once you submit the form.

**Reply by xfwil** [reply]
Yah, i have tried like this but still not solved.

**Comment by xfwil** [comment]
Solved, i do with manually dispatch event and clearly the input with .value javascript. Maybe this my template problems.
